packages/csharp/PortingAssistant/PortingAssistant.Telemetry/Utils/LogUploadUtils.cs

```diff
@@ -1,254 +1,73 @@
-using Amazon.Runtime;
-using Amazon.Runtime.CredentialManagement;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
 using PortingAssistantExtensionTelemetry.Model;
 using System;
-using System.Collections;
-using System.Collections.Generic;
 using System.IO;
 using System.Linq;
-using System.Net;
-using System.Net.Http;
 using System.Net.NetworkInformation;
-using System.Text;
-using System.Threading.Tasks;
 using PortingAssistant.Client.Telemetry;
-using Amazon;
+using Serilog;
 
 namespace PortingAssistant.Telemetry.Utils
 {
     public static class LogUploadUtils
     {
-        private static Boolean IsFileLocked(FileInfo file)
+        private static string GetLogName(string file)
```

Comment:
I don't fine invocation to this method except from tests, is it used in the code logic some where?

Reply:
it's a function we pass into the uploader, each of the tool/ide/cli had different logic to parse out the log name so this is the solution I came up with.